docs/index.html

```diff
@@ -1144,6 +1150,18 @@ <h2>Interface</h2>
         types of object.
       </p>
 
+      <p>
+        A wallet implementation MAY NOT implement all the interfaces defined below.
```

Comment:
"MAY NOT" may be read as forbidding, i.e., "you can't implement them all", so some change is necessary.  My wording may be not be the best  Other suggestions welcomed! 
```suggestion
        A wallet implementation MAY implement a subset of the interfaces defined below.
```

Reply:
@TallTed I agree, updated accordingly.